test/integration/docs_examples_multiple_providers.py

```diff
@@ -59,33 +59,36 @@ def _get_raw_key(self, key_id):
 
 
 def cycle_file(key_arn, source_plaintext_filename, botocore_session=None):
-    """Encrypts and then decrypts a file under both a KMS Master Key Provider and a custom static Master Key Provider.
+"""Encrypts and then decrypts a file using a KMS master key provider and a custom static master
```

Comment:
Spacing: The docstring for a function must be indented to the level of the contents of that function.